scripts/pylib/twister/twisterlib/testsuite.py

```diff
@@ -265,7 +287,7 @@ def scan_testsuite_path(testsuite_path):
     ztest_suite_names = []
 
     src_dir_path = _find_src_dir_path(testsuite_path)
-    for filename in glob.glob(os.path.join(src_dir_path, "*.c*")):
+    for filename in find_c_files_in(src_dir_path):
```

Comment:
doesn't this work as needed just with `recursive=True` and `**` ? (assuming the current requirement for the python version  > 3.8)
```suggestion
    for filename in glob.glob(os.path.join(src_dir_path, "**/*.c*", recursive=True)):
```

Reply:
`**/*.c*` also matches e.g. `prj.conf`. Maybe `{path}/**/*.c*` works 🤷‍♂️